application/hubspot/v0/config/tasks.json

```diff
@@ -0,0 +1,1149 @@
+        }
+      },
+      "required": ["email"],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Obtain contact ID",
+      "instillUIOrder": 0,
+      "properties": {
+        "contact-id": {
+          "description": "Contact ID",
+          "instillUIOrder": 0,
+          "required": [],
+          "title": "Contact ID",
+          "type": "string",
+          "instillFormat": "string"
+        }
+      },
+      "required": ["contact-id"],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_GET_DEAL": {
+    "instillShortDescription": "Get deal information using deal ID",
+    "input": {
+      "description": "Input deal ID",
+      "instillUIOrder": 0,
+      "properties": {
+        "deal-id": {
+          "description": "Input deal ID",
+          "instillAcceptFormats": ["string"],
+          "instillUIMultiline": true,
+          "instillUIOrder": 0,
+          "instillUpstreamTypes": ["value", "reference", "template"],
+          "title": "Deal ID",
+          "type": "string"
+        }
+      },
+      "required": ["deal-id"],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Deal information",
+      "instillUIOrder": 0,
+      "properties": {
+        "owner-id": {
+          "$ref": "#/$defs/common/owner-id",
+          "instillFormat": "string",
+          "required": []
+        },
+        "deal-name": {
+          "$ref": "#/$defs/deal/deal-name",
+          "instillFormat": "string",
+          "required": []
+        },
+        "pipeline": {
+          "$ref": "#/$defs/deal/pipeline",
+          "instillFormat": "string",
+          "required": []
+        },
+        "deal-stage": {
+          "$ref": "#/$defs/deal/deal-stage",
+          "instillFormat": "string",
+          "required": []
+        },
+        "amount": {
+          "$ref": "#/$defs/deal/amount",
+          "instillFormat": "number",
+          "required": []
+        },
+        "deal-type": {
+          "$ref": "#/$defs/deal/deal-type",
+          "instillFormat": "string",
+          "required": []
+        },
+        "create-date": {
+          "$ref": "#/$defs/deal/create-date",
+          "instillFormat": "string",
+          "required": []
+        },
+        "close-date": {
+          "$ref": "#/$defs/deal/close-date",
+          "instillFormat": "string",
+          "required": []
+        },
+        "associated-contact-id": {
+          "$ref": "#/$defs/common/associated-contact-id",
+          "instillUIorder": 9
+        }
+        
+      },
+      "required": ["deal-name", "pipeline", "deal-stage"],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_CREATE_DEAL": {
+    "instillShortDescription": "Create new deal",
+    "input": {
+      "description": "Deal information",
+      "instillUIOrder": 0,
+      "properties": {
+        "contact-id-or-email": {
```

Comment:
Mainly same answer as above. I want to give an easier option for the user, but also allow contact-id just in case email does not exist.

That property is used to associate contact(s) to a deal. I will change this to an array later on